### DISCUSSION TITLE
How to make an AMP page in Nuxt 3.

### DISCUSSION THREAD

**Discussion by Tyraxaes** [nuxt/nuxt#19857] [Questions]
As title sais, since @nuxtjs/amp seems to not work, how can we achieve a Nuxt3 AMP without interfering with the rest of the pages created.

**Comment by dhaladitya108** [comment]
@Tyraxaes did you figure out something?

**Reply by Tyraxaes** [reply]
Hello, sorry for not answering earlier.
We did a plugin to control by url our AMP pages, and in that plugin we call our own parser, we ended up making the replaces on our own.
